client/app.js

```diff
@@ -16,6 +16,13 @@
  * @fileoverview Basic configuration for the TIE application.
  */
 
+var globalData = {
+	// Question data will be stored here, keyed by question ID.
```

Comment:
indentation -- use spaces not tabs

(funny that linter didn't catch this -- I'll investigate)